### ISSUE TITLE
Android Studio keeps complaining AndroidManifest.xml error

### ISSUE THREAD

**Issue by harryhow** [google/beacon-platform#24]
When I import project file to Android Studio (v2.0) and keep seeing this error.
Any hint?

![screen shot 2015-12-10 at 1 45 30 pm](https://cloud.githubusercontent.com/assets/693098/11724812/52a47802-9f44-11e5-8730-a7b868ef0927.png)

**Comment by mashbridge** [comment]
are you importing as an existing project, or trying to import it into something else? it has to be as an existing project.

if not, have you edited anything? under "android application" where you have "Android Beacon" i think it should just say "app". (the string "Android Beacon" doesn't appear anywhere in the source which makes me think it's something you've changed.)

**Comment by harryhow** [comment]
Yes, i'm importing existing project. I added 'Android Beacon'  because of when I Run -> Run 'App', I see this

![screen shot 2015-12-10 at 3 31 37 pm](https://cloud.githubusercontent.com/assets/693098/11727985/367b734a-9f55-11e5-9e42-b7033fdbeeb5.png)

It looks like I have to add a configuration in order to build, so that I add 'Android Application' under the name of 'Android Beacon'. This is where I showed you. 

I also tried to start over again, clone a new repo from here and re-import from the existing project. Still no luck, thanks for helping out.

**Comment by mashbridge** [comment]
What version of android studio are you using?

**Comment by harryhow** [comment]
Android Studio 2.0 Preview 2
![screen shot 2015-12-10 at 6 57 21 pm](https://cloud.githubusercontent.com/assets/693098/11732206/ec13a8ca-9f6f-11e5-9dcf-ede76c8cccff.png)

**Comment by mashbridge** [comment]
ah, you're on the bleeding edge. i'll update my world here later today and see what's happening. i wouldn't be shocked if there has been a breaking change or two :-)

**Comment by harryhow** [comment]
Thanks @mashbridge. Let me know if you want to test something on my side.

**Comment by mashbridge** [comment]
I installed the latest 2.0 3b build (linux) of Studio from http://tools.android.com/download/studio/canary/latest and tried opening the BeaconServiceDemoApp project. After indexing for a while it failed because it couldn't load one of the .iml files. (I'm not too surprised the files are incompatible -- the build structure of 2.0 is very different and often enough a major version bump implies some level of backwards incompatibility.)

I closed studio, deleted the current build state with `cd BeaconServiceDemoApp && 
rm -rf .gradle local.properties .idea build captures *.iml app/*.iml app/build`, opened studio again and re-imported the project. Then it worked fine.

You may not need to blow away your entire state -- it may just be the .iml files that need to be deleted. But it seems easy enough to get things working now. Hope that helps!
